tests/unit/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM miyurud/jasminegraph:20230909T110050
```

Comment:
For future unit tests, they need to be run in a k8s pod in order to establish connection with the kubernetes API server.
This kind of thing
https://github.com/FYP-Auto-Scale-JasmineGraph/jasminegraph/pull/21/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R30

Reply:
Done